projects/1.18/assets/botania/botania/lang/zh_cn.json

```
@@ -1629,75 +1668,78 @@
   "botania.entry.welcome": "欢迎来到植物魔法世界",
   "botania.tagline.welcome": "欢迎光临植物魔法酒店",
   "botania.page.welcome0": "$(o)欢迎使用植物魔法！首先，我祝贺你做出了正确的决定-把一本书和树苗绑在一起，或者是偶然在什么箱子里找到了这本巨著。怎样都可以啦。$(p)$(o)你可以通过页面下方的箭头、鼠标滚轮或键盘上的方向键来浏览这本书。",
-  "botania.page.welcome1": "$(o)剥开其光鲜的外表，植物魔法的本质却是“科技Mod”，因为对于科技Mod的每一个环节来说，你都需要心灵手巧、深思熟虑还有红石电路，方能不断推进科技前进。将这句话铭记在心，会很有用的。$(p)$(o)当然啦，不要担心，你同时还会得到一些酷炫的新玩具、方块还有装备。",
-  "botania.page.welcome2": "$(o)不论出于什么意图或者目的，对于那些以建造好玩、新奇而酷炫的东西为初心的玩家来说，植物魔法都将会是他们的最爱。虽然它是以完全独立，即可以单独进行游戏的标准设计的，但同时它也能和其它Mod打成一片。$(p)$(o)除去极少数个案，植物魔法的一切都是可以仅靠Minecraft的特性以及Mod自身的特性就可以完成的。",
-  "botania.page.welcome3": "$(o)当你迷失前进的方向时，不妨去看下本Mod的进度和挑战列表，这两个列表都可以在本书首页的左侧找到。$(p)$(o)本书排版友好，同时也提供了大量方便浏览的功能，所以请在正式开始探究前稍微花点时间来了解这些小玩意儿都能做些什么。",
+  "botania.page.welcome1": "$(o)剥开其光鲜的外表，植物魔法的本质却是“科技模组”，因为对于科技模组的每一个环节来说，你都需要心灵手巧、深思熟虑、还有红石电路，方能不断推进科技前进。将这句话铭记在心，会很有用的。$(p)$(o)当然啦，不要担心，你同时还会得到一些酷炫的新玩具、方块还有装备。",

```

中文不在and之前加逗号，原文 `it requires you to put to use ingenuity, thought, and redstone to create and progress`
```suggestion
  "botania.page.welcome1": "$(o)剥开其光鲜的外表，植物魔法的本质却是“科技模组”，因为对于科技模组的每一个环节来说，你都需要聪明才智、细心思考和红石技术才能不断推进科技前进。将这句话铭记在心，会很有用的。$(p)$(o)当然啦，不要担心，你同时还会得到一些酷炫的新玩具、方块还有装备。",

```